src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,104 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+import { findParent } from '../utils/ParentLink';
+import { SearchLink } from './SearchLink';
+
+export type Props = {
+  people: Person[],
+  selectedPerson: string,
+  sort: string,
+  order: string,
+};
+const searchColumnParams = ['Name', 'Sex', 'Born', 'Died'];
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  selectedPerson = '',
+  sort,
+  order,
+}) => {
+  const getSortParams = (column: string) => {
```

Comment:
Change the initial order to 'asc' instead of null for better readability and to match the task description.